README.md

```diff
@@ -579,5 +581,9 @@ versions as the required `pip` versions.
 [buildstatus-gha-image]: https://github.com/jazzband/pip-tools/workflows/CI/badge.svg
 [codecov]: https://codecov.io/gh/jazzband/pip-tools
 [codecov-image]: https://codecov.io/gh/jazzband/pip-tools/branch/main/graph/badge.svg
+[Matrix Room Badge]: https://img.shields.io/matrix/pip-tools:matrix.org?label=Discuss%20on%20Matrix%20at%20%23pip-tools%3Amatrix.org&logo=matrix&server_fqdn=matrix.org&style=flat
+[Matrix Room]: https://matrix.to/#/%23pip-tools:matrix.org
+[Matrix Space Badge]: https://img.shields.io/matrix/jazzband:matrix.org?label=Discuss%20on%20Matrix%20at%20%23jazzband%3Amatrix.org&logo=matrix&server_fqdn=matrix.org&style=flat
+[Matrix Space]: https://matrix.to/#/%23jazzband:matrix.org
```

Comment:
```suggestion
[Matrix Room Badge]: https://img.shields.io/matrix/pip-tools:matrix.org?label=Discuss%20on%20Matrix%20at%20%23pip-tools%3Amatrix.org&logo=matrix&server_fqdn=matrix.org&style=flat
[Matrix Room]: https://matrix.to/#/#pip-tools:matrix.org
[Matrix Space Badge]: https://img.shields.io/matrix/jazzband:matrix.org?label=Discuss%20on%20Matrix%20at%20%23jazzband%3Amatrix.org&logo=matrix&server_fqdn=matrix.org&style=flat
[Matrix Space]: https://matrix.to/#/#jazzband:matrix.org
```
Maybe this fixes the linkcheck?

Reply:
No, linkcheck attempts to find anchors within web pages. We should probably make it ignore matrix URLs via `linkcheck_ignore` in `conf.py`.

Reply:
Should I do a PR, or will you manage alone  😀  ?